mock/py/mockbuild/backend.py

```diff
@@ -787,7 +789,15 @@ def get_command(mode):
                 # on re-doing it
                 mode += ['--noprep']
 
-        self.buildroot.doChroot(get_command(mode),
+        # when we used dynamic_buildrequires, we must let rpmbuild check deps
+        # becasue rpmbuild will call %genearte_buildrequires once again
+        # to actually add the BuildRequires to the SRPM metadata
```

Comment:
Can you separate sentences with capital words and periods?  That would help me to understand this and the following sentence.

Reply:
Sure thing